mpa/modules/models/heads/custom_fcn_head.py

```diff
@@ -0,0 +1,123 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import torch
+import torch.nn as nn
+from mmcv.runner import force_fp32
+
+from mmseg.models.builder import HEADS
+from mmseg.models.decode_heads.fcn_head import FCNHead
+from mmseg.models.builder import build_loss
+from mmseg.models.losses import accuracy, LossEqualizer
+from mmseg.ops import resize
+from mmseg.core import add_prefix
+from mpa.modules.utils.mask_utils import get_ignored_labels_per_batch
+
+@HEADS.register_module()
+class CustomFCNHead(FCNHead):
+    """Custom Fully Convolution Networks for Semantic Segmentation.
+    """
+
+    def __init__(self,
```

Comment:
As I looked at it, FCNHead inherits the BaseDecodeHead.  The __init__ contents of CustomFCNHead match the init contents of BaseDecodeHead, is there a reason why you overridding this function? (I think it will work well same without the __init__).

Reply:
You're right. I missed to remove the duplicated codes. I'll remove the contents in init. Thanks!

Reply:
Would you remove __init__() in this PR?